ocfweb/docs/docs/services/hpc.md

```diff
@@ -1,65 +1,115 @@
+**NOTE: We are in the process of trialing this service to users so that we can make the service as accommodating and secure as possible. This means that items concerning the service, including this documentation, are subject to change. We will do our best to keep everyone updated and notified of changes as they come.**
 
 ## Introduction
 
-At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. Our main HPC server is currently named `corruption`, however we also have plans to expand the cluster to maximize the use of the resources at our disposal.
+At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. We currently have one main HPC server, which handles all of our HPC jobs. However, have plans to expand the cluster to maximize the use of the resources at our disposal.
 
-### Gaining Access
+## Gaining Access
 
-In order to access the HPC cluster, you must send a proposal to [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu). Make sure to include your OCF username or group account name and a short, but detailed technical description of the experiments you plan to run on our HPC infrastructure. This would include information about the nature of the software being developed, as well as the amount of computational resources that are expected to be needed.
+In order to access the HPC cluster, please send an access request to [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu). Make sure to include your OCF username or [group account][group] name and a detailed technical description of the projects you plan to run on our HPC infrastructure. This would include information about the nature of the software being run, as well as the amount of computational resources that are expected to be needed.
 
-#### Connecting
+### Connecting
 
-Our Slurm master node is currently `segfault`. Once you submit your proposal and are approved access you will be able to connect via ssh by running the following command:
+Our Slurm master node is named `hpcctl`. Once you submit your proposal and are approved access you will be able to connect via ssh by running the following command:
 
-    ssh my_ocf_username@segfault.ocf.berkeley.edu
+```
+ssh my_ocf_username@hpcctl.ocf.berkeley.edu
+```
 
-If you have trouble connecting please email us at [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu).
+If you have trouble connecting please [contact us][contact] at [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu), or come to [staff hours][staff_hours] when the lab is open and chat with us in person. We also have a hpc_users [slack][fco] channel where you can ask questions and talk to us about anything HPC.
```

Comment:
> We also have a hpc_users [slack][fco] channel where you can ask questions and talk to us about anything HPC.

Perhaps suggest IRC instead/as well? That way they don't have to jump through hoops to sign up for a Slack account...

Reply:
Yeah. With the web interface it honestly might be easier for users. Also don't forget to mention the #hpc channel

Reply:
Maybe we should reserve #hpc for interal HPC discussion and ask users to stick to #hpc_users?

Reply:
Yeah I agree with abizer on this one